portal-frontend/src/app/features/applications/application-details/pofo-details/pofo-details.component.html

```diff
@@ -21,23 +21,12 @@
     <app-no-data [showRequired]="showErrors" *ngIf="!_applicationSubmission.purpose"></app-no-data>
   </div>
 
-  <div class="subheading2 grid-1">Project Duration</div>
+  <div class="subheading2 grid-1">Placement of Fill Project Duration</div>
   <div class="grid-double">
-    <div *ngIf="_applicationSubmission.soilProjectDurationAmount && _applicationSubmission.soilProjectDurationUnit">
-      {{ _applicationSubmission.soilProjectDurationAmount }}
-      <ng-container [ngPlural]="_applicationSubmission.soilProjectDurationAmount">
-        <ng-template ngPluralCase="=1">
-          {{ _applicationSubmission.soilProjectDurationUnit.slice(0, -1) | titlecase }}
-        </ng-template>
-        <ng-template ngPluralCase="other">
-          {{ _applicationSubmission.soilProjectDurationUnit | titlecase }}
-        </ng-template>
-      </ng-container>
+    <div *ngIf="_applicationSubmission.soilProjectDuration">
+      {{ _applicationSubmission.soilProjectDuration }}
```

Comment:
I am not too sure, I had questions about that too to be honest. However, since it was originally soilProjectDurationAmount and *Unit, I used soilProjectDuration here. This could've been before we had two types of project duration for soil submissions. Or maybe because when users switch between application types, we want the project duration fields to remain the same?

Reply:
Not a big deal, just wondering if it made sense. I'm not too concerned.

Reply:
If this field was going to be used in analytics / reporting I would be more concerned, but for now its all good.